addons/pos_self_order/static/tests/tours/test_self_order_kiosk.js

```diff
@@ -112,3 +113,17 @@ registry.category("web_tour.tours").add("self_simple_order", {
         Utils.checkIsNoBtn("My Order"),
     ],
 });
+
+registry.category("web_tour.tours").add("self_order_price_null", {
+    test: true,
+    steps: () => [
+        Utils.checkIsNoBtn("My Order"),
+        Utils.clickBtn("Order Now"),
+        ProductPage.clickProduct("Coca-Cola"),
+        Utils.clickBtn("Order"),
+        CartPage.checkProduct("Coca-Cola", "0.00", "1"),
+        Utils.clickBtn("Pay"),
+        ConfirmationPage.orderNumberShown(),
```

Comment:
If the order number is always the same (I guess it's always something like 0001) you could make sure that the correct number is displayed.

Reply:
It changes from a time to another. I tried twice, the first time the number was 701 and the second time it was 801